### DISCUSSION TITLE
Add a `GOOD_REASON` termination reason

### DISCUSSION THREAD

**Discussion by JSv4** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#145] [Suggestions] [answered]
**Per issue #56, importing comments / suggestions to our static Google Doc into GitHub for discussion**

@ashawber proposed we add a `GOOD REASON` enum to Termination Window type's `reason` field's enumerated reasons:

https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/9edd343c01303f82a2b3e5161faebab7637e27ad/schema/enums/TerminationWindowType.schema.json#L1-L15

@ashawber, are you proposing we add another reason type called `GOOD_REASON`, and, if so, could we roll it under `CAUSE`?

**Comment by ashawber** [comment]
Yeah I think we need another reason type for Good Reason. It can be rolled under "Voluntary" because it is a subtype of Voluntary termination.

Side note: Cause is a subtype of Involuntary termination.

**Comment by JSv4** [comment]
Question here - what is ultimately driving these different enumerations? Are these based purely on convention, or does this stem from some of the applicable SEC regs here? Would be helpful to get a complete breakdown of what needs to be covered here. @ashawber, are you able to add some color here?

**Comment by ashawber** [comment]
The “Cause” and “Good Reason” enumerations are relevant for the vesting of stock options, in particular acceleration of stock options upon a change of control. Knowing the circumstances upon which an option holder was terminated will often determine whether any vesting acceleration is triggered. These things are creatures of the contractual conventions around employee equity arrangements, which are often negotiated when an exec is hired.

Also, if an employee is terminated for “Cause,” then their vested options immediately terminate and are no longer exercisable. This is how most stock plans are set up to prevent bad actors from being able to exercise their vested options. Again, this is a contractual convention that is part of most stock plans.

On Mar 29, 2022, at 9:58 AM, JSIV ***@***.***> wrote:

﻿

Question here - what is ultimately driving these different enumerations? Are these based purely on convention, or does this stem from some of the applicable SEC regs here? Would be helpful to get a complete breakdown of what needs to be covered here. @ashawber<https://urldefense.com/v3/__https://github.com/ashawber__;!!K_R5ZAeIjLw!UXREHOUUqxbhh0WVPqH6nooyZmmshqq1bwCiIOMCCLSl207gPylqLDw2sR89qtqTFuc$>, are you able to add some color here?

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/discussions/145*discussioncomment-2463464__;Iw!!K_R5ZAeIjLw!UXREHOUUqxbhh0WVPqH6nooyZmmshqq1bwCiIOMCCLSl207gPylqLDw2sR89tWbrVsU$>, or unsubscribe<https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AX5MIGVBXCJ55UV5HU4OHELVCMZEHANCNFSM5R4EFUVA__;!!K_R5ZAeIjLw!UXREHOUUqxbhh0WVPqH6nooyZmmshqq1bwCiIOMCCLSl207gPylqLDw2sR89CnkEDW8$>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

________________________________

NOTICE: This communication may contain privileged or other confidential information. If you have received it in error, please advise the sender by reply email and immediately delete the message and any attachments without copying or disclosing the contents. Thank you.

**Comment by JSv4** [comment]
@ashawber, the original enumeration here, if I'm remembering correctly, was based on what the respective vendor platforms supported. We're open to tweaking the list, but, if we do so, we want to make sure it doesn't just add more options to pick from that people could be confused about. I want to make sure we have clarity about the distinctions between the options and the hierarchy (e.g. Cause is a subtype of involuntary - so do we track both? Do we only track the ultimate child type - i.e. Cause)?

Since this is based on the underlying SOPs, I think it would be great to get the proposed Standards Working Group to weigh in on what the "right" enumerations should be before making changes here (you should have just got an invite from David @ WSGR). That said, if you think we absolutely must change what's here before release because something critical is missing, mind taking a stab at what you think the enumeration should be for release?

**Comment by ashawber** [comment]
The original enumerations are much more confusing, actually, because there is overlap between them (e.g. Retirement is a subtype of Voluntary, and Cause is a subtype of Involuntary)

The enumerations should break down like this, which represent the various ways that a service provider can be terminated. It’s just two main types (Voluntary and Involuntary) and their respective sub-types:


  1.  Voluntary:
     *   Retirement
     *   Good Reason
     *   Other Voluntary (e.g. termination by the employee for a reason other than Retirement or “Good Reason”)
  2.  Involuntary:
     *   Cause
     *   Death
     *   Disability
     *   Other Involuntary (e.g. termination by the Company without Cause, and no Death or Disability)

Doing it this way allows the user to specify the circumstances under which a service provider terminated their relationship with the Company and eliminates any confusion.

Andy Shawber | Perkins Coie LLP
SENIOR INNOVATION COUNSEL
D. +1.206.359.3896 (rings through to cell)
E. ***@***.******@***.***>

**Reply by JSv4** [reply]
Thanks @ashawber. This makes sense to me. @Open-Cap-Table-Coalition/twg, any thoughts on this proposal?

**Reply by tmccnnll** [reply]
I have a few thoughts and questions about this.

- Instead of `GOOD_REASON`, as is being proposed, what if we used `GOOD_CAUSE`, which seems to be a widely used terminology across many US states ([for example](https://edd.ca.gov/en/uibdg/Voluntary_Quit_VQ_5/#:~:text=%22Good%20cause%22%20exists%20for%20leaving,leave%20work%20under%20the%20same))?
- Should we change `CAUSE`, a sub-variation of `INVOLUNTARY`, to `WITH_CAUSE` to be more explicit/verbose?
- Do we want to maintain a generic `VOLUNTARY` and `INVOLUNTARY` enum member while also supporting the more specific sub-variations of those members? 
  - This could support the "other" category instead of us having to constantly update this enum
- Semi-related to the above, should we prefix all of the sub-variations of the `VOLUNTARY` and `INVOLUNTARY` enum members with those two words to better show their relationships?

Taking the four bullet points into account, you would end up with:
```
"enum": [ 
     "VOLUNTARY",
     "VOLUNTARY_GOOD_CAUSE",
     "VOLUNTARY_RETIREMENT",
     "INVOLUNTARY", 
     "INVOLUNTARY_DEATH", 
     "INVOLUNTARY_DISABILITY", 
     "INVOLUNTARY_WITH_CAUSE",
   ] 
```

**Comment by JSv4** [comment, marked as answer]
**From the TWG Call on 4/12/22:**

We didn't want to have `involuntary` and `voluntary` top-level choices as we were worried people would default to those and omit additional detail they might and which would be helpful. Let's make them `voluntary_other` and `involuntary_other` to reflect that this isn't meant to be a hierarchy.

So, for completeness' sake, this would be the enum:

```
"enum": [ 
     "VOLUNTARY_OTHER",
     "VOLUNTARY_GOOD_CAUSE",
     "VOLUNTARY_RETIREMENT",
     "INVOLUNTARY_OTHER", 
     "INVOLUNTARY_DEATH", 
     "INVOLUNTARY_DISABILITY", 
     "INVOLUNTARY_WITH_CAUSE",
   ] 
```
